repository/user/user.go

```diff
@@ -41,6 +42,16 @@ func (colUser *Collection) InsertUser(ctx context.Context, user models.NewUser)
 		
 	}
 }
+//GetUserLogin check in datebase
+func (colUser *Collection) GetUserLogin(ctx context.Context, user models.UserLogin) (models.UserLogin, error) { 
```

Comment:
nazwalbym to GetUserByEmail i zwracal calego usera z bazy, moze sie przydac w innych miejscach.